packages/graphql/src/types/index.ts

```diff
@@ -507,11 +508,28 @@ export interface Neo4jFiltersSettings {
 export interface Neo4jPopulatedBySettings {
     callbacks?: Neo4jGraphQLCallbacks;
 }
+export interface Neo4jAuthorizationSettings {
+    key: Key | ((req: RequestLike) => Key);
+    verify?: boolean;
+    verifyOptions?: JWTVerifyOptions;
+}
+export interface RemoteJWKS {
+    url: string | URL;
+    options?: RemoteJWKSetOptions;
+}
+export type Key = string | RemoteJWKS;
+export type RequestLike = {
+    headers?: { authorization?: string; Authorization?: string };
+    rawHeaders?: Array<string>;
+    cookies?: { token?: string };
+};
 
 export interface Neo4jFeaturesSettings {
     filters?: Neo4jFiltersSettings;
     populatedBy?: Neo4jPopulatedBySettings;
+    authorization?: Neo4jAuthorizationSettings;
 }
+export { RemoteJWKSetOptions, JWTVerifyOptions } from "jose";
```

Comment:
I believe these will also need exporting in the root-level `index.ts` right? I'm not sure how others feel, but something like a `re-exports.ts` somewhere feels sensible to me, so we always have a good eye on what we're re-exporting rather than having them littered around.

Reply:
How about a new file in `types` directory called `external`? This way they are contained and it's easy to spot them.

Reply:
Sounds good!